Formula/php@8.1-phalcon.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "../lib/php_pecl_formula"
+
+class PhpAT81Phalcon < PhpPeclFormula
+  extension_dsl "Phalcon is a full stack PHP framework offering low resource consumption and high performance"
+
+  url "https://pecl.php.net/get/phalcon-5.9.0.tgz"
+  sha256 "7b8622bf7df5385e159dde3f41ed91bc6798726d8d725a46db8ba884651664d0"
+  license "PHP-3.01"
```

Comment:
Also important to advertise the proper license. I believe it's `BSD-3-Clause` from [here](https://github.com/phalcon/cphalcon/blob/60ab4914eb9282e289d9d51065674b38e5c85813/LICENSE.txt).

Reply:
Thanks. I fixed this locally but didn't push because I got distracted.